doc/API.md

```diff
@@ -57,7 +159,7 @@ struct {
 'target' is the hash that needs to be met by a block for the block to be valid.
 The target is inversely proportional to the difficulty.
 
-#### /consensus/block [GET]
+#### /consensus/blocks/:height [GET]
```

Comment:
another thing that occurred to me was how we should document path parameters. In this PR they are listed along with the URL parameters. I think that the distinction is fairly obvious, but perhaps it's not explicit enough.